plone-4.3.x.cfg

```diff
@@ -1,39 +0,0 @@
-[buildout]
```

Comment:
I see that `make update` is used to update the versions of this package. The `Makefile` file still tries to download cfgs from unsupported versions. It is necessary to remove these files from the `Makefile`:

https://github.com/plone/plone.restapi/blob/at-cleanup/Makefile#L27-L28

Reply:
@wesleybl thanks for the review! I am aware of this problem and I plan to put some effort into fixing it. Though, not in this PR.

Reply:
OK @tisto Just for the record, I created #1144